packages/babel-core/test/fixtures/transformation/source-maps/input-source-map-external/source-map.json

```diff
@@ -1,7 +1,11 @@
 {
-  "mappings": "AAAA,UAAU,Y;SAAM;AAAC,CAAjB",
+  "version": 3,
+  "mappings": "AAAA,UAAU,Y;SAAM,C;CAAC",
```

Comment:
This change is fine. The `;` is no longer mapped to a source location, which really don't matter.